goss-testing/tests/common/goss-network-interfaces.yaml

```diff
@@ -26,16 +26,18 @@
 {{ $logrun := .Env.GOSS_BASE | printf "%s/scripts/log_run.sh" }}
 command:
     {{range $interface := index .Vars "network_interfaces"}}
-        {{ $testlabel := $interface | printf "interface_%s" }}
-        {{$testlabel}}:
-            title: Interface '{{$interface}}' Has VLAN
-            meta:
-                desc: Ensures VLAN was created for network interface '{{$interface}}'.
-                sev: 0
-            exec: |-
-                "{{$logrun}}" -l "{{$testlabel}}" \
-                    ip link show dev "{{$interface}}"
-            exit-status: 0
-            timeout: 10000
-            skip: false
+        {{if ne $interface "bond0.can0"}}
```

Comment:
This is not what we want.   Some systems will have the bond0.can0 interfaces and some will not depending on whether CAN or CHN was configured when the system was set up.   I think we may want to have a script here that checks the BICAN network configuration in SLS.

Reply:
I'm fine with other options -- this is just the one that Jacob had implemented originally, in CASMINST-4196, and that I accidentally over-wrote in a PR. Right now the test fails on systems without bond0.can0, so unless we have a quick implementation of what you're suggesting, we may want to merge this PR if only as a stop-gap until we have a better solution.

If we do want to go that route, let me know, and I'll open another ticket to properly correct the test.

Reply:
I don't know if we will be doing many (or any) installs on wasp or hela in the next couple of days that would hit this issue again.   I feel like it will be better to have this tests the way it is to work on "CAN" systems (our more frequently installed systems) until we can get the fix that will address both "CAN" and "CHN" systems.

Reply:
Okie dokie, makes sense to me. I'll close these PRs and update the ticket to reflect its new scope.